Updates for GeoLadder

bolle732 opened the pull request:
Small upates

bluewaysw commented on Driver/Video/VGAlike/VGA16/vga16Tables.asm:
```diff
@@ -198,6 +200,8 @@ ifndef PRODUCT_WIN_DEMO
 		nptr    offset VidSetVESA		; VD_VESA_800x480_16
 		nptr    offset VidSetVESA		; VD_VESA_832x624_16
 		nptr    offset VidSetVESA		; VD_VESA_848x480_16
+		nptr    offset VidSetVESA		; VD_VESA_960x540_16
```
I think it is an leaning from the last weeks, that it is helpful to increase the "change" number of the geode version at least if some property of a geode changes, as new entries in the list of devices. This will cause the cache in the device driver list to be rerendered so the new entries become available automatically.

bluewaysw commented:
Do the changes to GeoLadder and the VGA driver neccessarily belong together? If not 2 pull request would be the preferred approach here. Not a blocker, maybe the next time.

bolle732 commented:
No, the new VGA modes are independent of the other changes. The problem here is, that Github add all new commits to the last pull request. A fast research didn't brought up an easy solution to split the pull request. What do you recommend?

bluewaysw commented:
Best would be to separate different changes on different branches, and have different PR for the branches. But leave it this time. Could you still update the rev file and increase the "change" version part of R-line by 1?

bolle732 commented:
Ok, I'll try to update the revision file.

The only thing is, that I want to understand that step. When looking into the current file, it looks like this at the beginning:

```
R 6.0.0.0 <bluewaysw> <15:25:29 Nov 19, 2023> <Prepare FreeGEOS 6.0 release.>
R 6.0.0.13 <frehwagen> <10:18:22 Nov 9, 2021> <bluewaysw>
R 6.0.0.12 <frehwagen> <17:34:19 Nov 8, 2021> <bluewaysw>
R 6.0.0.11 <frehwagen> <10:27:55 May 12, 2021> <bluewaysw>
R 6.0.0.10 <frehwagen> <20:03:19 Nov 22, 2020> <bluewaysw>
R 6.0.0.6 <frehwagen> <14:07:12 Nov 15, 2020> <bluewaysw>
R 6.0.0.1 <frehwagen> <15:25:54 Feb 2, 2020> <bluewaysw>
```

What is the reason, that the top most and by date newest entry has all 0 and the other older entries have a higher number then 0. Is this first line a special one?

I think, a new line like this should be the correct one:

`R 6.0.0.14 <bolle732> <10:00:00 Jan 2, 2025> <New video resolutions>`

I would use this commands:
```
$ cd $ROOT_DIR/Driver/Video/VGAlike/VGA16
$ grev neweng vga16.rev -s "New video resolutions"
```
Hope that is the correct.

bluewaysw commented:
Intersting, looks here I used 6.0.0.x base early (2020), later I update alle gedes to 6.0.0.0 again (2023). I think it uses the upmost. Anyhow the last number is the enginerring number, which is maintained now by Jenkins. The engineering number ist also ignored by the Preferences update mechanism. Because of that: Increment the "change" number, 2nd from right. You may use the tool or update the file using an file editor.

bolle732 commented:
Ok, I've update the rev file for the VGA16 and at the same case that of the GeoLadder too. What looks stange is the change in the vga16.rev. I've only added one line tot the rev, but git diff tells me, that I've removed and added all other lines too... Could this be a thing of text file encoding or line endings?
